frontend/src/components/Field/Rating/Rating.tsx

```diff
@@ -140,38 +147,27 @@ export const Rating = forwardRef<RatingProps, 'input'>(
           spacing={['0.5rem', '0.5rem', '0.5rem', '1rem']}
           align={['flex-start', 'flex-start', 'flex-start', 'center']}
         >
-          <Wrap spacing={ratingLayout.spacing}>
-            {options.map((value, i) => {
-              return (
-                <Fragment key={value}>
-                  <WrapItem>
-                    <RatingOption
-                      name={name}
-                      variant={variant}
-                      colorScheme={colorScheme}
-                      value={value}
-                      onChange={handleRatingChange}
-                      selectedValue={currentValue}
-                      isDisabled={isDisabled}
-                      {...(i === 0 ? { ref } : {})}
-                    >
-                      {value}
-                    </RatingOption>
-                  </WrapItem>
-                  {
-                    // Force component to begin on a new line.
-                    value !== numberOfRatings &&
-                      value % wrapComponentsPerRow === 0 &&
-                      isSplitRows && (
-                        <WrapItem display="contents" aria-hidden>
-                          <Box flexBasis="100%" h={ratingLayout.rowHeight} />
-                        </WrapItem>
-                      )
-                  }
-                </Fragment>
-              )
-            })}
-          </Wrap>
+          <Stack spacing={ratingLayout.spacing}>
+            {options.map((row, i) => (
+              <HStack spacing={ratingLayout.spacing} key={i}>
```

Comment:
slight regression here as seen in chromatic

Reply:
[8f1e49e](https://github.com/opengovsg/FormSG/pull/4908/commits/8f1e49ec278a06d8375289a2d745646e1ef5d10d)